### PR TITLE
Fix compile warnings in frontend

### PR DESCRIPTION
**Pull request by IidaKainu**

### CODE REVIEW

File: frontend/src/components/MissingRatings.tsx [hunk 1]
```diff
@@ -66,65 +66,61 @@ export const MissingRatings: FC<{
...
-  return (
-    <div className={classes(css.withLabel)}>
-      {missingDevRatings?.length || missingRatings?.length ? (
-        <Trans i18nKey="Waiting for ratings" />
-      ) : (
-        <Trans i18nKey="No ratings waiting" />
-      )}
-      <Icons />
+        {missingDevRatings &&
+          missingDevRatings.map(({ email }) => (
```
Comment (juhoeerola) [inline frontend/src/components/MissingRatings.tsx:113]:
```suggestion
        {missingDevRatings?.map(({ email }) => (
```

I don't think this was changed in this PR, but why not add it

File: frontend/src/components/UserInfoCard.tsx [hunk 2]
```diff
@@ -69,10 +69,13 @@ export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => {
...
-        <Trans i18nKey="Projects" />
-      </h2>
+    Title: useCallback(
+      () => (
+        <h2>
+          <Trans i18nKey="Projects" />
+        </h2>
+      ),
+      [],
     ),
```
Comment (juhoeerola) [inline frontend/src/components/UserInfoCard.tsx:79]: I'd rather just move this outside the `UserInfoCard` component

```suggestion
    Title,
```

```tsx
const Title = () => (
  <h2>
    <Trans i18nKey="Projects" />
  </h2>
);
```

**Review by juhoeerola — CHANGES_REQUESTED** [review]
Great 👍 
Just couple suggestions for simplifications

**Comment by IidaKainu** [comment]
> Just couple suggestions for simplifications

Added, thanks 👍

**Review by juhoeerola — APPROVED** [review]

**Pull request merged**